config.toml

```
@@ -1,3 +1,131 @@
 baseURL = "https://unicef.github.io/drone-dpgtoolkit"
 languageCode = "en-us"
 title = "Drone DPG Toolkit"
+theme = "inventory"
+enableEmoji = true
+enableGitInfo = true
+
+# Google Analytics
+googleAnalytics = "G-80FMX4TPM0"
+
+# Allow unsafe HTML
+[markup.goldmark.renderer]
+unsafe = true
+
+
+############################# Navigation #############################
+[[menu.main]]
+weight = 10
+name = "FAQ"
+url = "faq"
+
+[[menu.main]]
+weight = 20
+name = "pages"
+url = "pages"
+hasChildren = true
+
+
+
+
+############################# Default parameters #############################
+[params]
+# Need to replace with SVG logo
+#logo = "favicon.ico"
+
+# OpenGraph / Twitter Card metadata
+description = "UNICEF Open Source Inventory. A UNICEF Global Innovation knowledge base of best practices and resources for working and leading Open."
```

This is not relevant for the toolkit. Let's change this to the GitHub repository description, unless @enonied4g has another suggestion for what this should be:

```suggestion
description = "Toolkit of drones-for-public-good resources including text, image, video and audio content, and Open Source software, code, libraries, and training datasets."
```